internal/profile/profile.go

```diff
@@ -98,19 +108,20 @@ func Register(config *Config, router gin.IRouter) error {
 			return
 		}
 
+		version := time.Now().UnixNano() / int64(time.Millisecond)
 		_, err = db.Exec(`
-INSERT INTO profiles (user_id, profile) VALUES($1, $2)
+INSERT INTO profiles (user_id, profile, version) VALUES($1, $2, $3)
 ON CONFLICT (user_id)
-DO UPDATE SET profile = $2`,
-			userID, data)
+DO UPDATE SET profile = $2, version = $3`,
```

Comment:
the version should be updated automatically

Reply:
yes it is, i shouldn't push this, one thing about this, since the "version" is generated on a DB level, i will need to change the exec for something else, or query the db after the insert.

Reply:
mmm since this is a ON CONFLICT i'm not sure but you can return from an update or INSERT in psql by using RETURNING, if it doesn't work for you let me know and I can help you to research

Reply:
Yeap, i was not sure if `RETURNING ` would work, but it does. Thanks for the tip!